airflow/bin/cli.py

```diff
@@ -72,15 +72,30 @@ def sigquit_handler(sig, frame):
     print("\n".join(code))
 
 
-def setup_logging(filename):
-    root = logging.getLogger()
+def setup_file_logging(filename, fmt=settings.SIMPLE_LOG_FORMAT):
     handler = logging.FileHandler(filename)
```

Comment:
I was trying to keep this as simple as possible and still address the JIRA issue (Airflow doesn't play nicely with existing logger configurations).  I think this PR does accomplish that, and as is, should be beneficial to any Airflow users operating a more complicated logging setup.

Since I've posted this PR, my team has decided that Airflow is not appropriate for our use case, so I probably won't be able to spend any cycles updating it.  If you prefer, I can close it?